router/api.go

```diff
@@ -204,6 +209,109 @@ func (api *API) DeleteRoute(ctx context.Context, w http.ResponseWriter, req *htt
 	w.WriteHeader(200)
 }
 
+func (api *API) CreateCert(ctx context.Context, w http.ResponseWriter, req *http.Request) {
+	log, _ := ctxhelper.LoggerFromContext(ctx)
+
+	var cert *router.RouteCert
+	if err := json.NewDecoder(req.Body).Decode(&cert); err != nil {
+		log.Error(err.Error())
+		httphelper.Error(w, err)
```

Comment:
Probably is, the extra log statement is in all the other api handlers i looked at so i'll clean that up.